src/box/lua/console.lua

```diff
@@ -488,6 +501,12 @@ local text_connection_mt = {
                     end
                 end
             end
+            if operators[items[1]] == set_param and
+                param_handlers[items[2]] == set_continuation then
+
+                self.continuation_on = items[3] == 'on'
+            end
```

Comment:
The `\set output <...>` handling above do nothing on incorrect input: I guess it leans on server's response to report an error. Here the option is turned of on, say, `\set continuation xxx`.

Reply:
```
tarantool> require("console").connect("127.0.0.1:3311")
connected to 127.0.0.1:3311
---
- true
...

127.0.0.1:3311> \set c on
---
- error: 'Unknown parameter: c'
...

127.0.0.1:3311> \set continuation on
---
- true
...

127.0.0.1:3311> \set continuation xxx
---
- error: 'usage: \set continuation on|off'
...

127.0.0.1:3311> 
```

Reply:
> Here the option is turned off after, say, `\set continuation xxx`.

(I edited the phrase a bit to make it easier to understand.)

The example above neither approve, nor decline it.

Reply:
Ok, now I've got it. Will fix.

Reply:
Fixed.